features/frontend/product_reviews.feature

```diff
@@ -0,0 +1,73 @@
+@products
+Feature: Product reviews
+    In order to rate and learn more about the products
+    As a Visitor
+    I want to browse and post product reviews
+
+    Background:
+        Given there is default currency configured
+        And there are following taxonomies defined:
+            | code | name     |
+            | RTX1 | Category |
+        And there are following users:
+            | email       | password | enabled |
+            | bar@foo.com | foo1     | yes     |
+        And taxonomy "Category" has following taxons:
+            | Clothing[TX1] > PHP T-Shirts[TX2] |
+        And there are following channels configured:
+            | code   | name       | currencies | locales             | url       |
+            | WEB-EU | mystore.eu | USD        | en_GB, fr_FR, de_DE | localhost |
+        And the following products exist:
+            | name            | price | taxons       | pricing calculator | calculator configuration | attributes |
+            | Symfony T-Shirt | 15.00 | PHP T-Shirts |                    |                          |            |
+        And channel "WEB-EU" has following configuration:
+            | taxonomy |
+            | Category |
+        And channel "WEB-EU" has following products assigned:
+            | product         |
+            | Symfony T-Shirt |
+        And there are following reviews:
+            | title       | rating | comment                | author      | product         | subject type |
+            | Lorem ipsum | 5      | Lorem ipsum dolor sit  | bar@foo.com | Symfony T-Shirt | product      |
+            | Consectetur | 4      | Consectetur adipiscing | bar@foo.com | Symfony T-Shirt | product      |
+            | Proin nibh  | 3      | Proin nibh augue       | bar@foo.com | Symfony T-Shirt | product      |
+
+    Scenario: Displaying product rating and reviews:
+        Given I am on the store homepage
+        And I follow "PHP T-Shirts"
+        When I click "Symfony T-Shirt"
```

Comment:
Replace this 3 steps above with `Given I am on the product page for "Symfony T-Shirt`